### PR TITLE
Change Endpoint scoring files to point to model as a directory 

### PR DESCRIPTION
**Pull request by Chris113113**
Swaps all of the endpoint samples to use a directory-based model instead of a file-based model to align with the run_spec for custom_model being a uri_folder.

# PR into Azure/azureml-examples

## Checklist

I have:

- [x] read and followed the contributing guidelines

## Changes

-

fixes #

### CODE REVIEW

**Comment by rsethur** [comment]
Thanks for fixing this. Please ensure you touch all of the workflow files (in .github/workflows) and ensure they are ALL tested before merging this
image

**Comment by rsethur** [comment]
@Chris113113  you also need to add workflow files for deploy-local-endpoint and deploy-moe-autoscale.
Ignore the failure for uai - its a known issue and someone is fixing it (cli-scripts-deploy-managed-online-endpoint-access-resource-uai)

File: .github/workflows/cli-scripts-deploy-managed-online-endpoint-access-resource-sai.yml [hunk 1]
```diff
@@ -10,6 +10,8 @@ on:
       - cli/deploy-managed-online-endpoint-access-resource-sai.sh
       - .github/workflows/cli-scripts-deploy-managed-online-endpoint-access-resource-sai.yml
       - cli/setup.sh
+      - cli/endpoints/online/managed/managed-identity
```
Comment (rsethur) [inline .github/workflows/cli-scripts-deploy-managed-online-endpoint-access-resource-sai.yml:13]: did you mean this folder instead?: cli/endpoints/online/managed/managed-identities

File: .github/workflows/cli-scripts-deploy-managed-online-endpoint-access-resource-uai.yml [hunk 2]
```diff
@@ -10,6 +10,8 @@ on:
       - cli/deploy-managed-online-endpoint-access-resource-uai.sh
       - .github/workflows/cli-scripts-deploy-managed-online-endpoint-access-resource-uai.yml
       - cli/setup.sh
+      - cli/endpoints/online/managed/managed-identity
```
Comment (rsethur) [inline .github/workflows/cli-scripts-deploy-managed-online-endpoint-access-resource-uai.yml:13]: did you mean this folder instead?: cli/endpoints/online/managed/managed-identities

File: .github/workflows/cli-scripts-deploy-managed-online-endpoint-mlflow.yml [hunk 3]
```diff
@@ -10,6 +10,8 @@ on:
       - cli/deploy-managed-online-endpoint-mlflow.sh
       - .github/workflows/cli-scripts-deploy-managed-online-endpoint-mlflow.yml
       - cli/setup.sh
+      - cli/endpoints/online/managed
```
Comment (rsethur) [inline .github/workflows/cli-scripts-deploy-managed-online-endpoint-mlflow.yml:13]: this does not seem to be required for mlflow sample

File: .github/workflows/cli-scripts-deploy-managed-online-endpoint.yml [hunk 4]
```diff
@@ -10,6 +10,8 @@ on:
       - cli/deploy-managed-online-endpoint.sh
       - .github/workflows/cli-scripts-deploy-managed-online-endpoint.yml
       - cli/setup.sh
+      - cli/endpoints/online/managed
```
Comment (rsethur) [inline .github/workflows/cli-scripts-deploy-managed-online-endpoint.yml:13]: Needs to be this: endpoints/online/managed/sample
The above path lots of stuff not needed by this script

File: .github/workflows/cli-scripts-deploy-safe-rollout-online-endpoints.yml [hunk 5]
```diff
@@ -10,6 +10,9 @@ on:
       - cli/deploy-safe-rollout-online-endpoints.sh
       - .github/workflows/cli-scripts-deploy-safe-rollout-online-endpoints.yml
       - cli/setup.sh
+      - cli/endpoints/online/managed
```
Comment (rsethur) [inline .github/workflows/cli-scripts-deploy-safe-rollout-online-endpoints.yml:13]: Needs to be this: endpoints/online/managed/sample
The above path lots of stuff not needed by this script

**Review by rsethur — CHANGES_REQUESTED** [review]

**Review by rsethur — CHANGES_REQUESTED** [review]

**Review by rsethur — APPROVED** [review]

**Review by raghutillu — APPROVED** [review]

**Pull request merged**